test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -3560,6 +3560,17 @@ def test_layer_norm(self):
         x = torch.randn(20, 5, 10, 10)
         self.run_test(model, x)
 
+    @skipIfUnsupportedMaxOpsetVersion(14)
```

Comment:
I think testing that it successfully exports on opset_version 15 is necessary and sufficient (i.e., replace this test with an opset_version 15 test that doesn't fail).

Reply:
Makes sense. Can you please add some of this to a doc string of the test method?